packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -46,7 +46,7 @@ export class DataBuilder {
     dataElementCodes,
     additionalQueryConfig,
     aggregationType = this.aggregationType,
-    aggregationConfig = this.config.aggregationConfig,
+    aggregationConfig = this.config.aggregationConfig ?? {},
```

Comment:
Awesome, didn't know about this operator! Dev chat?

Reply:
I think I got it from dev chat!!! Nick mentioned it I believe

Reply:
Oh shows how much attention I pay...I'm gonna claim being away for that dev chat 😅